doc/changelog/08-vernac-commands-and-options/19981-classify-warn.rst

```diff
@@ -0,0 +1,5 @@
+- **Fixed:**
+  :opt:`Debug` and :opt:`Warnings` are classified as Synterp.
```

Comment:
It's a shame that https://github.com/coq/coq/pull/19759 is not merged yet, otherwise we could refer to it here.

Reply:
Cc @Zimmi48 FYI (but definitely not a serious/urgent issue)